LMS/Billings/Services/Gerencianet/GerencianetService.php

```diff
@@ -83,7 +83,8 @@ public function createSubscription($planId, string $name, float $price): array
                     ]
                 ],
                 'metadata' => [
-                    'notification_url' => route('billing-callbacks', ['provider' => 'gerencianet'])
+                    'notification_url' => 'https://requestinspector.com/inspect/01ffrpsj1nyxdx8k9k177ftphb',
+//                    'notification_url' => route('billing-callbacks', ['provider' => 'gerencianet']),
```

Comment:
Caso for realmente utilizar esta linha comentada, este gerenciamento de notificação é interessante aplicar abstração na composição